### ISSUE TITLE
feat(Data): Note in profile page that Data represents Published Expeditions

### ISSUE THREAD

**Issue by irag** [BillionOysterProject/Digital-Platform#595]
This is an enhancement.  There is no bug.

My BOP user name is Ira Gershenhorn.

I took the following screenshot of the result of filtering Community>People by gershenhorn and then clicking on the image of the one returned person (me).  Its obvious to me that Data represents Published Expeditions and Launched Expeditions represent expeditions NOT published.  I realize that an expedition only becomes Data once its been published but it just strikes me that there's this syntactic change.  I would prefer if Data was followed by (Published Expeditions) and Launched Expeditions were followed by (expeditions not published), because to my mind the expeditions listed under Data certainly were launched.

![image](https://user-images.githubusercontent.com/808169/34347235-16bec586-e9cf-11e7-86b7-d4599f7a13a8.png)

The file where this appears seems to be `\modules\profiles\clients\views\view-user-content.client.view.html`
in Lines 91, 105, 150, 162 (there are two cases of Data and Launched Expeditions).

There seems to be two different presentations based on whether one is the TeamLead or not.

If I could I would tag this as enhancement and help wanted.

**Comment by haflanagan** [comment]
@irag This is a good point- I can see how it's not super clear what represents unpublished vs. published data.  This is something I can code myself so I'll play around with this and double check with the team on their preferred language and I'll let you know what we come up with!

**Comment by irag** [comment]
@haflanagan Yes.  Definitely get agreement on what the language should be.  I must have added the parenthetical part only because I wanted to limit my change. So maybe instead of Data it should be something like Published Expeditions used as Data and the second group might be Unpublished Expeditions.  I'm not sure what 'launched' means. Either there's an expedition or there's not.

**Comment by haflanagan** [comment]
@irag I checked with the team- we're going to go with "Launched (Unpublished)" and "Published."  This was a great suggestion!  (BTW, "launched" means an expedition was created, and you're right, technically every expedition on the platform has been launched, both published and unpublished expeditions.)  Thanks so much for the feedback!  I'll ping you when this change goes through (I'll probably get to it next week).

**Comment by irag** [comment]
@haflanagan Fabulous! I know its so hard to get a decision like this.
